Post/Audio/Get audio working in Hackintosh.md

```diff
@@ -0,0 +1,29 @@
+# Get audio working in Hackintosh
```

Comment:
how about rename this file as `README.md`? So it can be shown in a more obviously way.